drm-howto/modeset-atomic.c

```diff
@@ -0,0 +1,1082 @@
+/*
+ * modeset-atomic - DRM Double-Buffered Atomic-API Modesetting Example
+ *
+ * Based on:
+ *
+ * modeset - DRM Double-Buffered VSync'ed Modesetting Example
+ *
+ * Written 2012 by David Herrmann <dh.herrmann@googlemail.com>
```

Comment:
Is it?

Reply:
How to change `Written 2012 by David ...` to `Written 2019 by Ezequiel ...` while giving credit to the original author?

Reply:
@leandrohrb Can you squash both commits and remove the WIP from the commit log?
This header could be something like:

 modeset-atomic - DRM Atomic-API Modesetting Example
 Written 2019 by Ezequiel Garcia <ezequiel@collabora.com>

 Based on:
 modeset-double-buffered.c - DRM Double-Buffered VSync'ed Modesetting Example
 Written 2012 by David Herrmann <dh.herrmann@googlemail.com>